templates/nav.hbs

```diff
@@ -14,7 +14,7 @@
     <span class="icon"></span>
   </button>
 
-  <button class="btn-icon nav__btn nav__drawer-btn js-nav-drawer-btn" data-event="toggleDrawer" aria-label="{{_globals._accessibility._ariaLabels.navigationDrawer}}" data-order="{{_globals._extensions._drawer._navOrder}}">
+  <button class="btn-icon nav__btn nav__drawer-btn js-nav-drawer-btn is-position-{{_config._drawer._position}}" data-event="toggleDrawer" aria-label="{{_globals._accessibility._ariaLabels.navigationDrawer}}" data-order="{{_globals._extensions._drawer._navOrder}}">
```

Comment:
Isn't this handled in the View className() already?

Reply:
No, that's for the child view of the drawer, this is the navigation button.